src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './NewCommentForm.scss';
+import { addPostComment } from '../../api/comments';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
-    >
-      Add a comment
-    </button>
-  </form>
-);
+export const NewCommentForm = ({ postId }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (name && email && body) {
+      addPostComment(postId, name, email, body);
+
+      setName('');
+      setEmail('');
+      setBody('');
+    }
+  };
+
+  return (
+    <form className="NewCommentForm" onSubmit={handleSubmit}>
+      <div className="form-field">
+        <input
+          type="text"
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+          value={name}
+          onChange={({ target }) => setName(target.value.trim())}
```

Comment:
```suggestion
          onChange={({ target }) => setCommentPart(target.name, target.value)}
```
create method setCommentPart which should work like this:
```javascript
const setCommentPart = (key, value) => {
  setComment(prevComment => ({
    ...prevComment,
    [key]: value,  
  })
}
```